core/scripts/sql/texera_ddl.sql

```diff
@@ -19,10 +19,11 @@ SET GLOBAL time_zone = '+00:00'; # this line is mandatory
 
 CREATE TABLE IF NOT EXISTS user
 (
-    `name`      VARCHAR(32)                 NOT NULL,
-    `uid`       INT UNSIGNED AUTO_INCREMENT NOT NULL,
-    `password`  VARCHAR(256),
-    `google_id` VARCHAR(256) UNIQUE,
+    `name`       VARCHAR(32)                 NOT NULL,
+    `uid`        INT UNSIGNED AUTO_INCREMENT NOT NULL,
+    `password`   VARCHAR(256),
+    `google_id`  VARCHAR(256) UNIQUE,
+    'permission' ENUM('pending', 'approved', 'admin') DEFAULT 'pending',
```

Comment:
This `permission` is actually for authentication purpose, so I feel it could also just be `role`, which is corresponding to the `javax.annotation.security.Roles`. 

I suggest we change it to 
```suggestion
    'role' ENUM('none', 'user', 'admin') DEFAULT 'none',
```

or 

```suggestion
    'role' ENUM('inactive', 'user', 'admin') DEFAULT 'inactive',
```

what do you think?

Reply:
changed